File: DB.MSSQL/DB.v4.MS_SQL/Interfaces/ISQLServerDBPointer.vb
Imports DB.v4.MS_SQL

Namespace Interfaces

    ''' <summary>
    ''' Mainly for pointing to a Specific Server Instance
    ''' </summary>
    ''' <remarks></remarks>
    Public Interface ISQLServerDBPointer

        Function getDBConn() As Server


    End Interface


End Namespace

File: EEntityCore.DB/EEntityCore.DB/Schemas/SQLServerSchema/ISQLServerDBPointer.cs

using EEntityCore.DB.Interfaces;

namespace EEntityCore.DB.Schemas.SQLServerSchema
{

    /// <summary>
    /// Mainly for pointing to a Specific Server Instance
    /// </summary>
    /// <remarks></remarks>
    public interface ISQLServerDBPointer
    {

        /// <summary>
        /// Gets a new instance of SQL Server Connection
        /// </summary>
        /// <returns></returns>
        IAll__DBs GetAllDBConn();
    }
}